### DISCUSSION TITLE
How to make v_slot in edit dialog within datatable?

### DISCUSSION THREAD

**Discussion by zhang9song** [Kitware/trame#241] [Q&A] [answered]
Hi, guys,

I want to make a editable data table, How to write the v_slot like  **v-slot:item.name="props"**?
Thanks for your reply.
vuetify example: [https://v2.vuetifyjs.com/en/components/data-tables/#edit-dialog](url)

with vuetify code like:
   ```
<v-data-table
      :headers="headers"
      :items="desserts"
    >
      <template v-slot:item.name="props">
        <v-edit-dialog
          :return-value.sync="props.item.name"
        >
          {{ props.item.name }}
          <template v-slot:input>
            <v-text-field
              v-model="props.item.name"
              :rules="[max25chars]"
              label="Edit"
              single-line
              counter
            ></v-text-field>
          </template>
        </v-edit-dialog>
      </template>
    ```

**Comment by jourdain** [comment, marked as answer]
`v-slot:item.name="props"` => `v_slot_item_name="props"`

That part is explained in [the course reference guide](https://github.com/Kitware/trame-tutorial/raw/master/course/introduction/trame_intro_ref.pdf).

That information was also provided here: https://github.com/Kitware/trame/discussions/227

**Reply by zhang9song** [reply]
Thanks for your help.
